lib/evss/disability_compensation_form/data_translation_all_claim.rb

```diff
@@ -86,15 +92,44 @@ def output_form
       end
 
       def overflow_text
-        return nil unless @has_form4142 || input_form['isTerminallyIll'].present?
-
         overflow = ''
         overflow += TERMILL_OVERFLOW_TEXT if input_form['isTerminallyIll'].present?
         overflow += FORM4142_OVERFLOW_TEXT if @has_form4142
 
+        if Flipper.enabled?(:form526_include_document_upload_list_in_overflow_text)
+          overflow += FORM0781_OVERFLOW_TEXT if @form_content.key?(Form526Submission::FORM_0781)
+          overflow += attached_files_list
+        end
+
         overflow
       end
 
+      def attached_files_list
+        list = ''
+        attachments = input_form['attachments']
+
+        if attachments&.length
```

Comment:
Could you also just check for the existence of attachments or if there are any attachments?

Reply:
I wish, our database isn't normalized this way, the attachment relationship for a form submission is nested in this JSON field which drives me banana sandwiches. Good comment though

Reply:
I may have misunderstood your question here but this behavior got refactored in https://github.com/department-of-veterans-affairs/vets-api/pull/16551/commits/1a5822abd337d1c2aab155365e016aab2b03bd44 so let me know if you spot anything I may have missed